examples/typescript/testDBConnections.ts

```diff
@@ -0,0 +1,56 @@
+import {LookerNodeSDK} from '@looker/sdk-node'
+import {IDBConnection, IDBConnectionTestResult} from '@looker/sdk'
+import { DelimArray } from '@looker/sdk-rtl'
+
+
```

Comment:
Formatting? Lots of whitespace. lint/prettier?

Reply:
Cleared whitespace in the latest commits. John is going to help me run lint/prettier checks on these before merged